toffy/image_utils.py

```diff
@@ -0,0 +1,26 @@
+import skimage.io as io
+import pathlib
+import numpy as np
+from typing import Union
+
+
+def save_image(fname: Union[str, pathlib.Path], data: np.ndarray,
```

Comment:
Nice, make sense to default all of the check_contrast flags into one function. This should get added to the list of things to put in TMI